pkg/storagev2/storagetest/storage.go

```diff
@@ -10,14 +10,50 @@ import (
 	"encoding/binary"
 	"encoding/json"
 	"errors"
-	"reflect"
+	"fmt"
 	"strconv"
 	"strings"
 	"testing"
 
 	"github.com/ethersphere/bee/pkg/storagev2"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/google/go-cmp/cmp"
 )
 
+var (
+	// MinAddressBytes represents bytes that can be used to represent a min. address.
+	MinAddressBytes = [swarm.HashSize]byte{swarm.HashSize - 1: 0x00}
+
+	// MaxAddressBytes represents bytes that can be used to represent a max. address.
+	MaxAddressBytes = [swarm.HashSize]byte{0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF}
+)
+
+var _ storage.Item = (*ItemStub)(nil)
+
+// ItemStub is a stub for storage.Item.
+type ItemStub struct {
```

Comment:
Is this required?

Reply:
Yes, for cases where you want to simulate an inappropriate buffer size, e.g. for Item Unmarshal.